packages/contracts-core/.gas-snapshot

```diff
@@ -0,0 +1,26 @@
+HomeTest:test_homeDomain() (gas: 8010)
```

Comment:
was this file intentionally committed? should it be?

Reply:
Yes. It's a nice little trick I learned from t11s. By having snapshots of the gas usage, you can easily see if some change in the code results in c change of the gas consumption since the tests remain the same.